start-servant.cabal

```diff
@@ -8,23 +8,7 @@ extra-source-files:
 common common-extensions
   default-language: Haskell2010
   default-extensions:
-    DeriveGeneric
-    DerivingVia 
-    FlexibleContexts
-    FlexibleInstances
-    GADTs
-    InstanceSigs
-    LambdaCase
-    MultiParamTypeClasses
-    NamedFieldPuns
-    OverloadedStrings
-    PackageImports
-    RecordWildCards
-    ScopedTypeVariables
-    TupleSections
-    TypeApplications
-    ViewPatterns
-    TypeApplications
+    StrictData 
```

Comment:
- I knew about `Strict`, but not `StrictData`, nice !
- Just as I think we should mention `Protolude` in the README, we could also mention `StrictData`.